types/testing/transaction.go

```diff
@@ -1,11 +1,11 @@
 package tt
 
 import (
-	"time"
-
 	"github.com/mitchellh/go-testing-interface"
-	pbaptos "github.com/streamingfast/firehose-aptos/types/pb/aptos/extractor/v1"
+	"github.com/streamingfast/firehose-aptos/types/pb/aptos/extractor/v1"
 	pbtimestamp "github.com/streamingfast/firehose-aptos/types/pb/aptos/util/timestamp"
+	"time"
```

Comment:
Not sure why imports are re-ordered here, conventions is that native Golang are put first then a blank line than all others.